pkg/build/info_test.go

```diff
@@ -298,13 +297,12 @@ func TestUnmarshalInfo(t *testing.T) {
 			name:  "unmarshal string",
 			input: "an string value",
 			expected: &Info{
-				Name: "an string value",
+				Context: "an string value",
```

Comment:
Same, why are we replacing Name by Context ? should it be Image?

Reply:
This is the test for the previous scenario:
```yaml
services:
  api:
    build: api
```
We are passing the context. We were using name to later set the real context